setup.py

```diff
@@ -46,7 +46,8 @@
     'websockets',
     'appdirs',
     'urllib3<1.25',
-    'tqdm'
+    'tqdm',
+    'aenum'
```

Comment:
IMO, we should conditionally add `aenum` for Python versions < 3.4

Reply:
Actually `aenum` is required for every python version as the new Enum implementation doesn't support nested enums. We're trying to keep things as close to original puppetteer's code as possible. See related: https://bugs.python.org/issue33976

Reply:
Ahh, I see